kubernetes/ansible/roles/logging/templates/fluentd-elasticsearch.yaml

```diff
@@ -48,3 +48,82 @@ extraConfigMaps:
       </buffer>
     </match>
     </label>
+
+  containers.input.conf: |-
```

Comment:
You have to disable the default configmap in the dictionary  before this gets applied. Check this line